scripts/trivia.coffee

```diff
@@ -48,8 +48,8 @@ module.exports = (robot) ->
     
   #solve attempt matches anything after !, case insensitive
   #need to add some way to score points
-  robot.respond /!(.+)$/i, (msg) ->
-    solveAttempt = msg.match[1]
+  robot.respond /!\s?(.+)$/i, (msg) ->
```

Comment:
If you make the regex to /! \s+(. +)/i it will match any random number of spaces.